src/exam/Exam.jsx

```diff
@@ -25,9 +25,7 @@ const Exam = ({
 }) => {
   const state = useContext(ExamStateContext);
   const {
-    isLoading, activeAttempt, showTimer, stopExam, exam,
-    expireExam, pollAttempt, apiErrorMsg, pingAttempt,
-    getProctoringSettings, submitExam,
+    isLoading, showTimer, exam, apiErrorMsg, getProctoringSettings,
```

Comment:
Same issue with showTimer here

Reply:
Although maybe it's not an issue? I just am not sure why we need ExamStateContext anymore

Reply:
Just reiterating our conversation from standup for posterity, but I was just trying not to interfere with the files you were working on. I agree that we can move `showTimer` into the components that need it and get rid of `ExamStateProvider`!